src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
        int index = .nextInt(Colors.values().length);
```
lets make random global, this way you will create just 1 instance of this class instead of a new one each time this method will be invoked